cert/CACreationScript.sh

```diff
@@ -108,25 +108,33 @@ openssl ca -create_serial -cert $PI.crt -keyfile $PI.pkey -outdir $CSR -batch -o
 #PAL: TODO
 
 #Device Intermediate
-CBSD=wif_cbsd_ca
+CBSD=cbsd_ca
 CBSD_SIGN=$CBSD$SIG
 
-openssl req -new -newkey rsa:$INT_RSA_KEY_SIZE -nodes -reqexts $CBSD -out $CBSD.csr -keyout $CBSD.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA Device CA-1" -config openssl.cnf
+openssl req -new -newkey rsa:$INT_RSA_KEY_SIZE -nodes -reqexts $CBSD -out $CBSD.csr -keyout $CBSD.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA CBSD CA-1" -config openssl.cnf
 
 openssl ca -create_serial -cert $ROOT.crt -keyfile $ROOT.pkey -outdir $CSR -batch -out $CBSD.crt -utf8 -days $INT_VALIDITY -policy policy_anything -md sha384 -extensions $CBSD_SIGN -config openssl.cnf -in $CBSD.csr
 
 #Device EE
-CBSD_EE=wif_cbsd_req
+CBSD_EE=cbsd_req
 CBSD_EE_SIGN=$CBSD_EE$SIG
 
-openssl req -new -newkey rsa:$EE_RSA_KEY_SIZE -nodes -reqexts $CBSD_EE -out $CBSD_EE.csr -keyout $CBSD_EE.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=Device End-Entity Example" -config openssl.cnf
+openssl req -new -newkey rsa:$EE_RSA_KEY_SIZE -nodes -reqexts $CBSD_EE -out $CBSD_EE.csr -keyout $CBSD_EE.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=CBSD End-Entity Example" -config openssl.cnf
 
 openssl ca -create_serial -cert $CBSD.crt -keyfile $CBSD.pkey -outdir $CSR -batch -out $CBSD_EE.crt -utf8 -days $EE_VALIDITY -policy policy_anything -md sha384 -extensions $CBSD_EE_SIGN -config openssl.cnf -in $CBSD_EE.csr
 
-#TODO: manufacturer intermediate signed by $CBSD, and EE cert signed by manufacturer intermediate
-
+#Device OEM Intermediate
```

Comment:
Do you think we should have a separate intermediat e for OEM EE? Or just use the device intermediate? My mental image was to just use the cbsd_ca for this. So IOW there'd still be the OEM cert (which should have ca:true and keydepth=1 in the profile), but it would be off a cbsd_ca intermediate. You think that's sufficient separation?